concatenate_flows/docker-run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+IMAGE_NAME=maap-concatenate_pdms
+
+# Check that the correct number of arguments were provided.
+if [ $# -ne 76 ]; then
```

Comment:
How did you manage to test this code without this line failing?

Reply:
It didn't, hadn't tested it yet

Reply:
Now tested